typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -691,6 +691,46 @@ export const gnosis: ChainMetadata = {
   gnosisSafeTransactionServiceUrl: 'https://safe-transaction.xdai.gnosis.io/',
 };
 
+export const solanadevnet: ChainMetadata = {
+  chainId: 13375,
```

Comment:
I think we said we'd use ID's like the one Alchemy uses right? https://github.com/hyperlane-xyz/hyperlane-warp-ui-template/pull/29/files#diff-aadd348c4fc9e8e97be7734d6b4bb7fcb97e350b90d8121a1042a19714618e50R10

Reply:
Planning to change these next week or so - I deployed these mailboxes before that. There are some small changes to the program code that will probably happen that I have to do before redeploying

Reply:
Ah sorry I see what you mean on chainId vs domainId, will change